tempesta_fw/http.c

```diff
@@ -831,8 +879,13 @@ tfw_http_req_fwd_send(TfwSrvConn *srv_conn, TfwServer *srv,
 	if (tfw_connection_send((TfwConn *)srv_conn, (TfwMsg *)req)) {
 		TFW_DBG2("%s: Forwarding error: conn=[%p] req=[%p]\n",
 			 __func__, srv_conn, req);
-		tfw_http_req_error(srv_conn, req, equeue, 500,
-				   "request dropped: forwarding error");
+		if (req->flags & TFW_HTTP_HMONITOR) {
+			tfw_http_req_delist(srv_conn, req);
+			tfw_http_msg_free((TfwHttpMsg *)req);
+		}
+		else
+			tfw_http_req_error(srv_conn, req, equeue, 500,
+					   "request dropped: forwarding error");
```

Comment:
Curly braces are required here according to the coding standard (because the other branch has them).

Reply:
Corrected.